Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Migration/Tools/MigrationWindow.cs

```diff
@@ -0,0 +1,417 @@
+                {
+                    migrationTypesMap.Add(migrationHandlerType.Name, migrationHandlerType);
+                }
+            }
+            migrationTypeNames = new String[migrationTypesMap.Count + 1];
+
+            for (int i = 0; i < migrationTypesMap.Count; i++)
+            {
+                migrationTypeNames[i + 1] = migrationTypesMap.ElementAt(i).Key;
+            }
+        }
+
+        private void DrawHeader()
+        {
+            MixedRealityInspectorUtility.RenderMixedRealityToolkitLogo();
+
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                EditorGUILayout.LabelField("Migration Window", EditorStyles.boldLabel);
+                InspectorUIUtility.RenderDocumentationButton(DependencyWindowURL);
+            }
+            EditorGUILayout.LabelField(WindowDescription, EditorStyles.wordWrappedLabel);
+
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+        }
+
+        private void DrawMigrationTypeSelector()
+        {
+            using (var horizontal = new EditorGUILayout.HorizontalScope())
+            {
+                EditorGUILayout.LabelField("Migration Handler Selection", EditorStyles.boldLabel);
+                selectedMigrationHandlerIndex = EditorGUILayout.Popup(selectedMigrationHandlerIndex, migrationTypeNames, GUILayout.Width(400));
+            }
+
+            if (selectedMigrationHandlerIndex == 0)
+            {
+                GUI.enabled = false;
+            }
+        }
+
+        private void DrawMigrationToolbars()
+        {
+            using (new GUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                EditorGUILayout.Space();
+
+                ToolbarOption previousSelectedToolbar = selectedToolbar;
+                selectedToolbar = (ToolbarOption)GUILayout.Toolbar((int)selectedToolbar, toolbarTitles);
+
+                if (previousSelectedToolbar != selectedToolbar)
+                {
+                    scrollPosition = Vector2.zero;
+                    migrationObjects.Clear();
+                }
+
+                if (selectedToolbar == ToolbarOption.Project)
+                {
+                    DrawProjectToolbar();
+                }
+                else
+                {
+                    DrawObjectsToolbar();
+                }
+            }
+        }
+
+        private void DrawObjectsToolbar()
+        {
+            EditorGUILayout.Space();
+
+            using (new GUILayout.HorizontalScope())
+            {
+                string tooltip = $"Drag and Drop {toolbarTitles[(int)selectedToolbar]} for Migration.";
+                EditorGUILayout.LabelField(new GUIContent($"{toolbarTitles[(int)selectedToolbar]} Selection", InspectorUIUtility.InfoIcon, tooltip));
+
+                var selectionType = selectedToolbar == ToolbarOption.Objects ? typeof(GameObject) : typeof(Object);
+                var allowSceneObjects = selectedToolbar == ToolbarOption.Objects;
```

Comment:
This line confuses me a little bit because I would assume the right side would be ToolbarOption.Scenes (maybe worth leaving a comment why this is objects and not scenes?)

Reply:
adding comment. Toolbar.Objects refering to Game Objects and Prefabs. (Maybe should change tab title to GameObjects?)